co-simulation tool/bridge/carla_integration/sumo_simulation.py

```diff
@@ -457,6 +459,19 @@ def tick(self):
         self.spawned_actors = set(traci.simulation.getDepartedIDList())
         self.destroyed_actors = set(traci.simulation.getArrivedIDList())
 
+        # Show v2x messages received by CARLA vehicles
+        v2xMessageReceived = traci.getV2xMessage()
+        if v2xMessageReceived is not None:
+            for message in v2xMessageReceived:
+                print(message)
+
+         # Send V2x message to CARLA ambassador
+        if self.sendV2xInterval == 10:          
```

Comment:
Here 10 means every 10 time-step. I assume that CARLA vehicles send V2X messages every 10 time steps. It can be set as any integer depending on users needs.